pyproject.toml

```diff
@@ -31,6 +31,7 @@ dependencies = [
   "httpx~=0.23.0",
   "orjson~=3.8.0",
   "click~=8.1.0",
+  "tenacity==8.2.2"
```

Comment:
Small library I'm pulling in for easily adding retry logic beyond what `httpx` allows. This is used down here - https://github.com/synccomputingcode/syncsparkpy/pull/18/files#diff-a92790f88cf634b038c61957033a487fc111749ec671d4821d0ac692fb69a94fR114

(`httpx` actually recommends this library in their docs as well - https://www.python-httpx.org/advanced/#usage_1)